tests/context.py

```
@@ -380,6 +380,108 @@
 	return [*args]
 
 
+def taint_command_args(args):
```

Ruff (0.8.2)

Missing return type annotation for public function `taint_command_args`

(ANN201)